lib/src/client.dart

```diff
@@ -1168,7 +1168,7 @@ class TwitchClient {
     final data = await twitchHttpClient.postCall<Map<String, dynamic>>(
       ['clips'],
       <String, Object>{},
-      queryParameters: {'broadcaster_id': broadcasterId, 'has_delay': hasDelay},
+      queryParameters: {'broadcaster_id': broadcasterId, 'has_delay': hasDelay.toString()},
```

Comment:
To avoid this kind of issue in the future I might update `queryParameters` to only accept a `Map<String, String>` 🤔